bootstrap/run_app_action.go

```diff
@@ -81,6 +81,12 @@ func (a *RunAppAction) Run(s *State) error {
 	interpolateRelease(s, a.Release)
 
 	for _, p := range a.Resources {
+		// Register provider without allocating resource.
```

Comment:
This change doesn't seem to be used? All of the resources specified in the manifest have `URL` set.